.github/workflows/update.yml

```diff
@@ -31,70 +31,153 @@ jobs:
+      id: redhat-2013
       run: ./vuln-list-update -target redhat -years 2013
+      continue-on-error: true
 
     - name: Red Hat Security Data API 2014
+      id: redhat-2014
       run: ./vuln-list-update -target redhat -years 2014
+      continue-on-error: true
 
     - name: Red Hat Security Data API 2015
+      id: redhat-2015
       run: ./vuln-list-update -target redhat -years 2015
+      continue-on-error: true
 
     - name: Red Hat Security Data API 2016
       run: ./vuln-list-update -target redhat -years 2016
+      continue-on-error: true
 
     - name: Red Hat Security Data API 2017
+      id: redhat-2017
       run: ./vuln-list-update -target redhat -years 2017
+      continue-on-error: true
 
     - name: Red Hat Security Data API 2018
+      id: redhat-2018
       run: ./vuln-list-update -target redhat -years 2018
+      continue-on-error: true
 
     - name: Red Hat Security Data API 2019-2020
+      id: redhat-2019-2020
       run: ./vuln-list-update -target redhat -years 2019,2020
+      continue-on-error: true
+
+    - name: Check for failures and Send a message to Microsoft Teams
+      uses: luisghz/simple-ms-teams-webhook-notifier@v1-latest
+      if: >
+        steps.nvd.outputs.status != 'success' ||
+        steps.alpine.outputs.status != 'success' ||
+        steps.debian.outputs.status != 'success' ||
+        steps.debian-oval.outputs.status != 'success' ||
+        steps.ubuntu.outputs.status != 'success' ||
+        steps.amazon.outputs.status != 'success' ||
+        steps.oracle-oval.outputs.status != 'success' ||
+        steps.redhat-oval.outputs.status != 'success' ||
+        steps.photon.outputs.status != 'success' ||
+        steps.ghsa.outputs.status != 'success' ||
+        steps.cwe.outputs.status != 'success' ||
+        steps.suse-cvrf.outputs.status != 'success' ||
+        steps.redhat-1996-2002.outputs.status != 'success' ||
+        steps.redhat-2003-2008.outputs.status != 'success' ||
+        steps.redhat-2012.outputs.status != 'success' ||
+        steps.redhat-2013.outputs.status != 'success' ||
+        steps.redhat-2014.outputs.status != 'success' ||
+        steps.redhat-2015.outputs.status != 'success' ||
+        steps.redhat-2016.outputs.status != 'success' ||
+        steps.redhat-2017.outputs.status != 'success' ||
+        steps.redhat-2018.outputs.status != 'success' ||
+        steps.redhat-2019-2020.outputs.status != 'success'
+      with:
+        webhook_url: ${{ secrets.AQUA_TEAMS_WEBHOOK }}
+        summary: 'vuln-list-update failed'
+        title: 'vuln-list-update'
+        text: 'One or more vendor advisory updates have failed. Check link for details.'
+        theme-color: 'Error'
+        potential-action: |
+          - "@type": "OpenUri"
+            name: "View Failures"
+            targets:
+                  - os: "default"
+                    uri: "https://github.com/aquasecurity/vuln-list-update/actions?query=is%3Afailure+event%3Aschedule"
```

Comment:
Recently Photon fails every day due to their issue. I'm afraid this notification would be noisy. What do you think? Even if we notify it, we need a separate channel.

Reply:
I'm fine with a separate channel.